internal/appsec/waf_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/DataDog/appsec-internal-go/apisec"
 	"github.com/stretchr/testify/assert"
 
+	"github.com/stretchr/testify/mock"
```

Comment:
Moving this seems unrelated... And the blank spacing there is odd... I'd normalize it or not touch it...